.github/workflows/ci.yml

```diff
@@ -140,6 +149,12 @@ jobs:
           # run cypress test cases
           npx cypress run --record --key ${{ secrets.CYPRESS_RECORD_KEY }}
 
+      - name: Coveralls
+        uses: coverallsapp/github-action@master
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
is this a new environment variable?

Reply:
nope. this is built-in with GitHub actions. Coveralls uses this to find the integrated github repository at it's end and upload the codecov into the right project.